docs/DeveloperGuide.md

```diff
@@ -153,6 +153,24 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Adding Items and Delivery
+OneShelf is capable of adding items and deliveries.
+Adding Items and Delivery both are done similarly which will be illustrated with an AddItemActivityDiagram below.
+
+![AddItemActivityDiagram](images/AddItemActivityDiagram.png)
+
+We've repurposed the `Add` command from AB3 to add on to existing item's quantity instead of throwing an error.
+In this case 2 items are considered the same if they have the same `Name` and `Supplier`.
+The other variables such as:
+
+a) `Metric` `MaxQuantity`
+- Are not allowed to be defined if there exist the same item inside InventoryBook.
+
+b) `Tags`
+- Will be combined together if there exist the same item inside InventoryBook.
+
+Note: No 2 deliveries are considered the same. Reason being the same person can make multiple delivery orders.
```

Comment:
No 2 can sound like number 2. And I don't really understand what is the message you're trying to bring across, perhaps a paraphrasing is needed?

Reply:
Paraphrased!